src/NServiceBus.AcceptanceTesting/packages.config

```diff
@@ -1,5 +1,6 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
-  <package id="GitVersionTask" version="3.6.5" targetFramework="net452" developmentDependency="true" />
+  <package id="GitVersionTask" version="4.0.0-beta0012" targetFramework="net452" developmentDependency="true" />
   <package id="NuGetPackager" version="0.6.1" targetFramework="net452" developmentDependency="true" />
+  <package id="UtilPack.NuGet.MSBuild" version="2.6.0" targetFramework="net452" developmentDependency="true" />
```

Comment:
What is this package? Why has it been added everywhere?

Reply:
this has been added when I updated GitVersionTask.

Reply:
Double checked: seems like this is a dependency from GitVersionTask beta14 which I accidentally updated to before downgrading to beta12 again. Let me try to remove that dependency again.